### DISCUSSION TITLE
پیشنهادات برای پایتونستان های بعدی

### DISCUSSION THREAD

**Discussion by seyedmm** [seyedmm/pythonestan#4] [Ideas]
<div dir=rtl>

# به این گپ خوش اومدید!👋👋
اینجا هر ایده ای دارید برای پایتونستان های بعد میتونید بنویسید. حتما بررسی و در صورت امکان منتشرشون میکنم😉

**Comment by radiumatic** [comment]
من یه موزیک پلیر مبتنی بر مرورگر جدیدا نوشتم که یه وب سرور کوچیک flask ایجاد می کنه و باهاش میشه به جای رابط کاربری از طریق مرورگر پخش رو کنترل کرد ولی فعلا سادس و فقط پخش و قطع و آدرس دهی رو پشتیبانی می کنه

**Comment by seyedmm** [comment]
<div dir=rtl>
عالیه! منتها من فلسک بلد نیستم |: میتونی کدشو بدی؟

**Comment by amirghazi87** [comment]
آقا یک پخش کننده موزیک ساده با کیو تی دیزاینر

**Reply by seyedmm** [reply]
<div dir=rtl>

عالیه! اگه تونستم بعد از [TTS](https://github.com/seyedmm/pythonestan/tree/TTS) پیاده‌ش میکنم. البته شاید بجای Qt از TkInter استفاده کردم
